### PR TITLE
Fe 7: QuizView with QuizSession

### PR DESCRIPTION
**Pull request by siffravn**

### CODE REVIEW

**Review by dethurah — APPROVED** [review]

File: web/src/hooks/useColors.ts [hunk 1]
```diff
@@ -1,10 +1,20 @@
 import { useColorModeValue } from "@chakra-ui/react";
+import { generateKeyPair } from "crypto";
```
Comment (dethurah) [inline web/src/hooks/useColors.ts:2]: what is this? :D

File: web/src/hooks/useColors.ts [hunk 2]
```diff
@@ -1,10 +1,20 @@
...
 
 export const useColors = () => {
+  const headerBg = useColorModeValue("gray.200", "gray.700");
+  const optionBg = useColorModeValue("gray.200", "gray.700");
+  const optionBgHover = useColorModeValue("gray.300", "gray.800");
+  const optionCorrect = useColorModeValue("green.400", "green.600");
+  const optionIncorrect = useColorModeValue("red.400", "red.600");
+    const primaryColorHover = useColorModeValue("gray.100", "generateKeyPair.700");
```
Comment (dethurah) [inline web/src/hooks/useColors.ts:10]: what color is this "generateKeyPair.700?

**Review by dethurah — CHANGES_REQUESTED** [review]

File: web/src/hooks/useColors.ts (hunk 1 above)
Comment (siffravn) [reply web/src/hooks/useColors.ts:2]: It is something that came with @magnusRBrok's changes in PR #22

File: web/src/hooks/useColors.ts (hunk 2 above)
Comment (siffravn) [reply web/src/hooks/useColors.ts:10]: It is something that came with @magnusRBrok's changes in PR #22

**Pull request merged**